pyproject.toml

```diff
@@ -1,13 +1,140 @@
 [build-system]
 requires = [
-    "setuptools >= 48, <60",
+    "setuptools >= 61",
     "setuptools_scm[toml] >= 4, <6",
     "setuptools_scm_git_archive",
     "milksnake",
     "wheel >= 0.29.0",
 ]
 build-backend = 'setuptools.build_meta'
 
+[project]
+name = "sourmash"
+description = "tools for comparing DNA sequences with MinHash sketches"
+readme = "README.md"
+
+authors = [
+  { name="Luiz Irber", orcid="0000-0003-4371-9659" },
+  { name="Mohamed Abuelanin", orcid="0000-0002-3419-4785" },
+  { name="Harriet Alexander", orcid="0000-0003-1308-8008" },
+  { name="Abhishek Anant", orcid="0000-0002-5751-2010" },
+  { name="Keya Barve", orcid="0000-0003-3241-2117" },
+  { name="Colton Baumler" },
```

Comment:
Hi @ccbaumler! Can you please provide your ORCID for our records? =]

Reply:
Hi Luiz,
I had created a branch with the ORCID ID you requested, but I am unable to `push` my `commit` upstream of your branch. I am assuming that is because this is a forked repository I do not have access to, is that correct?

Anyway, my ORCID is 0000-0002-5926-7792. 
Best,
Colton

Reply:
@mr-eyes helped me out with a suggestion tutorial. (https://www.youtube.com/watch?v=D8Drb2OKibs&ab_channel=AndreaMussap)

See the suggestion below.